math/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/clojure:tools-deps-bullseye
+FROM docker.io/clojure:temurin-17-tools-deps
```

Comment:
```suggestion
FROM docker.io/clojure:temurin-21-tools-deps
```

Reply:
unfortunately the issue reappears when theyre both 21

Reply:
@tevko  Can you explain why you want to indicate the _wrong_ version that you are _not_ using in math system properties?

Also I built and ran the images successfully on Mac M3 using v21 in both files. So your issue might be something else.

Reply:
see https://github.com/compdemocracy/polis/pull/1874